src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ifs/InterestForecastingSteps.scala

```diff
@@ -362,4 +362,13 @@ class InterestForecastingSteps extends ScalaDsl with EN with Eventually with Mat
   Given("debt payment plan details") { (dataTable: DataTable) =>
     createPaymentPlanRequestBody(dataTable)
   }
+
+  Then("the ([0-9])(?:st|nd|rd|th) debt summary will not have any calculation windows") { (summaryIndex: Int) =>
```

Comment:
We are still discussing this if this is the right way to go and need to discuss with Helen and Chris about this specific scenario.